docusaurus/docs/reference/ha/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_label: Overview
+sidebar_position: 05
+---
+
+# Controller HA 
+
+## Overview
+
+Ziti controllers can be run in a cluster for high availablity and performance scaling.
+
+:::warning
+
+**NOTE: Controller HA is still in Beta** 
+
+It's quite functional now, but we are continuing to test and refine before we mark it GA.
+:::
+
+
+### For SDK Clients/Tunnelers
+
+A controller cluster offers the following advantages:
+
+1. Horizontal scaling of SDK client services such as
+    1. Service lookups
+    1. Session creation
+1. Horizontal scaling of circuit creation
+
+This means that for everything that SDK clients and tunnelers depend on, controllers
+can be scaled up and placed strategically to meet user demand. 
+
+The following limitations currently apply:
+
+1. Circuits are owned by a controller. If the controller goes down, the circuit 
+   will remain up, but can't be re-routed for performance or if a router goes down.
+2. For a controller to route circuits on a router, that router must be connected
+   to that controller. This means that routers should generally be connected to
+   all controllers.
+
+### For Management Operations
+
+The HA controller cluster uses a distributed journal keep the data model synchronized across controllers. 
+This has the following ramifications:
+
+1. Read operations will work on any controller that is up. If the controller is 
+   disconnected from the cluster, the reads may return data that is out of date.
+2. Update operations require that the cluster has a leader and that a quorum of nodes
+   is available. A quorum for a cluster of size N is (N/2)+1. This means that a 3 node 
```

Comment:
does this mean it's impossible to run "HA" with a single (1) node? I realize one node isn't 'HA', but what I want to know here is if, as user, i can start with a single node or not. and if i start with a single node what ramifications will this have (if any). I am reading the doc and commenting as I go so if this is addressed elsewhere, a link to that might be useful here and something to the effect that yes you can run HA with one node makes sense to see here, imo...

Reply:
Also this might be just better as a link to some external doc. Regardless if you keep the text or not, a link to the doc is a good idea here

Reply:
Clarified that a one node will work